### PR TITLE
Updated title of text box in edit collection page. Fixes #5395

### LINKED ISSUE #5395: Edit collection page - The title of the text box is "User comment" instead of "Add-on note"
STR:
1. Load AMO dev and go to a collection
2. Click on Edit this collection and try to make a comment for an extension/theme from the list
3. Notice that the name of the text box is "User comment"
4. Save the comment and then click on Save collection
5. Observe the name of the text box

Expected result:
In both edit and view collection - "Add-on note" is displayed - as mentioned[ here]( https://user-images.githubusercontent.com/6797635/33636526-6ef3ae78-d9d1-11e7-9a80-5ce5bf85717b.png
)

Actual result:
There's an inconsistency between the two pages - Edit collection displays "User comment" instead of "Add-on note".

Notes:
This issue is reproducible on AMO dev with FF60(Win10). 
add-on note2

### PR DESCRIPTION
**Pull request by Shubhamchinda**
Fixes #5395 

In View mode :

Screenshot (163)

In Edit mode : 

Screenshot (164)

### CODE REVIEW

**Comment by willdurand** [comment]
@Shubhamchinda you should run `yarn prettier-dev` because one of the Travis CI jobs is failing: https://travis-ci.org/mozilla/addons-frontend/jobs/558857418

**Comment by Shubhamchinda** [comment]
Hey, @willdurand , Done!

File: src/amo/components/EditableCollectionAddon/index.js [hunk 1]
```diff
@@ -147,7 +147,9 @@ export class EditableCollectionAddonBase extends React.Component<InternalProps>
           <div className="EditableCollectionAddon-notes">
             <h4 className="EditableCollectionAddon-notes-header">
               <Icon name="comments-blue" />
-              {i18n.gettext('User comment')}
+              {this.props.uiState.editingNote
+                ? i18n.gettext('Leave a Note')
+                : i18n.gettext("Collector's Note")}
```
Comment (willdurand) [inline src/amo/components/EditableCollectionAddon/index.js]: Could you please lower case "Note" in the two strings?

**Comment by Shubhamchinda** [comment]
@willdurand I have a question. If you see the above images, the position of the "Remove" button changes when we click on the "Edit" button. It shouldn't happen, right?

**Comment by willdurand** [comment]
> @willdurand I have a question. If you see the above images, the position of the "Remove" button changes when we click on the "Edit" button. It shouldn't happen, right?

I see and that does not look nice, indeed.

**Comment by Shubhamchinda** [comment]
@willdurand should I try to fix it?

**Comment by willdurand** [comment]
> @willdurand should I try to fix it?

I believe it exists with or without this patch, right? If so, then we'll need an issue and another patch, but sure :)

**Comment by Shubhamchinda** [comment]
> > @willdurand should I try to fix it?
> 
> I believe it exists with or without this patch, right? If so, then we'll need an issue and another patch, but sure :)

Yeah, this issue was there before this patch. If you're creating an issue for this, just assign me! I'll be happy to work on it.

**Pull request merged**